tests/test_wrappers/test_sklearn_wrapper.py

```diff
@@ -414,6 +418,15 @@ def test_sklearn_ohe_with_crossvalidation():
     assert not any([np.isnan(i) for i in results])
 
 
+def test_wrap_one_hot_encoder_get_features_name_out(df_vartypes):
+    ohe = OneHotEncoder()
+    ohe_wrap = SklearnTransformerWrapper(transformer=OneHotEncoder(sparse=False))
+    ohe.fit(df_vartypes)
+    ohe_wrap.fit(df_vartypes)
+
+    assert (ohe.get_feature_names_out() == ohe_wrap.get_feature_names_out()).all()
```

Comment:
This is the test for OneHotEncoder

Reply:
Same as per previous comment, the idea is to just test the functionality of the method, and not in comparison to that of sklearn's.